src/Repository/SpecialBenefit/SpecialBenefitRepository.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository\SpecialBenefit;
+
+use App\Entity\SpecialBenefit;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class SpecialBenefitRepository extends ServiceEntityRepository implements SpecialBenefitRepositoryInterface
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, SpecialBenefit::class);
+    }
+
+    /**
+     * @param mixed    $id
+     * @param int|null $lockMode
+     * @param int|null $lockVersion
+     * @return SpecialBenefit|null
+     */
+    public function find($id, $lockMode = null, $lockVersion = null): ?SpecialBenefit
```

Comment:
Are we using both optional parameters anywhere? If not, just remove them.

Reply:
Oh... that's because we extend `ServiceEntityRepository` rather than injecting it... No worries, forget this comment!